docs/source/models/intervention_models/neonatal/antibiotics_intervention.rst

```diff
@@ -0,0 +1,131 @@
+
+We will use the decision tree below to find the probability of sepsis mortality with and without access to antibiotics that are logically consistent with the baseline delivery facility rates and baseline antibiotics coverage (and population level sepsis mortality probability). Since the neonatal model includes different mortality probabilities for both the early and late neonatal periods, this calculation must be performed for both time periods as well.
+
+We will then add an attribute to each simulant indicating whether the birth occurs at a facility with access to antibiotics (which will be dependent on the facility choice, i.e. no access for home births and lower access for BEmONC than CEmONC facilities).
+
+We will then use the conditional probabilities for simulants with and without access to determine which simulants die from sepsis.
+
+While we are searching the literature, we will use a placeholder relative risk of 0.50 (95% CI 0.25-0.75) of sepsis mortality for neonates with access to antibiotics.
+
+
+Calibration Strategy
+--------------------
+
+The following decision tree shows all of the paths from delivery facility choice to antibiotics availability.  Distinct paths in the tree correspond to disjoint events, which we can sum over to find the population probability of sepsis mortality.  The goal here is to find internally consistent probabilities of sepsis mortality for the subpopulations with and without access to antibiotics, so that the baseline scenario can track who has access to antibiotics and still match the baseline sepsis mortality rate.
+
+.. graphviz::
+
+    digraph antibiotics {
+        rankdir = LR;
+        facility [label="Facility type"]
+        home [label="p_sepsis_without_antibiotics"]
+        BEmONC [label="antibiotics"]
+        CEmONC [label="antibiotics"]
+        BEmONC_wo [label="p_sepsis_without_antibiotics"] 
+        BEmONC_w [label="p_sepsis_with_antibiotics"]
+        CEmONC_wo [label="p_sepsis_without_antibiotics"] 
+        CEmONC_w [label="p_sepsis_with_antibiotics"]
+
+        facility -> home  [label = "home birth"]
+        facility -> BEmONC  [label = "BEmONC"]
+        facility -> CEmONC  [label = "CEmONC"]
+
+        BEmONC -> BEmONC_w  [label = "available"]
+        BEmONC -> BEmONC_wo  [label = "unavailable"]
+
+        CEmONC -> CEmONC_w  [label = "available"]
+        CEmONC -> CEmONC_wo  [label = "unavailable"]
+    }
+
+.. math::
+    \begin{align*}
+        p(\text{sepsis}) 
+        &= \sum_{\text{paths without antibiotics}} p(\text{path})\cdot p(\text{sepsis_wo})\\
+        &+ \sum_{\text{paths with antibiotics}} p(\text{path})\cdot p(\text{sepsis_w})\\[.1in]
+        p(\text{sepsis_w}) &= \text{RR}_\text{antibiotics} \cdot p(\text{sepsis_wo})
```

Comment:
Once again, the $LaTeX$ is bad because of the underscores inside the `\text{}` commands (it renders on Read the Docs, but not on GitHub).

Reply:
Huh, TIL! I guess you need `\_` (or on GitHub `\\_` since the backslash also needs to be escaped).

$$\text{sepsis\\_w}$$